uptane-standard.md

```diff
@@ -867,11 +867,19 @@ In order to perform full verification, an ECU SHALL perform the following steps:
     1. If the top-level Targets metadata file contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
     2. Recursively search the list of delegations, in order of appearance:
-        0. If the delegation does not address the image name being sought -- if none of the delegation's image paths match this image name -- skip this delegation.
-        1. If it is a multi-role delegation {{TAP-3}}, recursively visit each role, and check that each has signed exactly the same non-custom metadata (i.e., length and hashes) about the image. If it is all the same, return the metadata to be checked and skip to step 11.3.
-        2. If it is a terminating delegation and it contains signed metadata about the image, return the metadata to be checked and skip to step 11.3. If metadata about an image is not found in a terminating delegation, return an error code indicating that the image is missing.
-        3. Otherwise, continue processing the next delegation, if any. As soon as a delegation is found that contains signed metadata about the image, return the metadata to be checked and skip to step 11.3.
-        4. If no signed metadata about the image can be found anywhere in the delegation graph, return an error code indicating that the image is missing.
+        1. If the delegation does not address the image name being sought -- if none of the delegation's image paths match this image name -- skip this delegation.
+        2. If it is a multi-role delegation {{TAP-3}}, then:
+            1. First, recurse (11.2) for each role listed as if each is a single-role delegation.
+            2. Inspect the return values from each:
+                1. If at least a number of roles equal to the minimum number of roles in agreement for this multi-role delegation return the same non-custom metadata (i.e. length and hashes) for the image, then return that metadata for the image.  If the values returned are such that there are multiple sets of non-custom metadata meeting the threshold (e.g. if the threshold is 2/4 and two sets of two roles agree on different metadata), then break the tie in favor of the earliest role in the multi-role delegation.
+                2. If the threshold for the minimum number of roles in agreement is not met:
+                    1. If this multi-role delegation is terminating, abort the search for this image, ending the recursion and returning an error code indicating that verified image metadata could not be found.
```

Comment:
For the terminating cases, I suggest replacing the comma after "abort the search for this image" with the word "by".

Reply:
Fixed, thanks.